src/components/Dropdown/DropdownItem.tsx

```diff
@@ -26,14 +28,21 @@ class DropdownItem extends React.Component<Props, never> {
 
   public id = getUniqueID();
 
+  clickCallback = (event: any) => {
+    const { onClick, returnValue } = this.props;
```

Comment:
@ankitsaraogi we are not actually looking at the component's callback function name but we do taking care of the props function name like it should be `onClick`

Reply:
@dheeraja00 @ankitsaraogi It would be nice to have consistency in component names as well.  It's not a big deal, but consistency is always a good to strive for.